harmony/harmony.py

```diff
@@ -802,8 +802,10 @@ def submit(self, request: BaseRequest) -> any:
         if response.ok:
             if isinstance(request, CapabilitiesRequest):
                 return response.json()
+            elif response.json()['status'] == 'successful':
+                return response.json()['links']
```

Comment:
We should return the full JSON response for the no op service. When there's a job the user can use the job ID to pull back the full job status information, but in this case there's no way to get the other information with another call. I'd want to see the message for example that the download links do not reflect the subsetting I requested.